CONTRIBUTING.md

```diff
@@ -19,9 +19,45 @@ We heartily welcome pull requests. To ensure an effective contribution, please a
 1. For significant, invasive, or output-affecting changes, consider opening an issue for discussion before committing substantial time and effort.
 2. If you're new to the project, starting with the [Good first issues](https://github.com/ocaml-ppx/ocamlformat/issues?utf8=%E2%9C%93&q=is%3Aissue+is%3Aopen+label%3A%22Good-first-issue+%3Agreen_heart%3A%22) can be beneficial.
 3. Fork the repository and create your branch from `main`.
-4. If you've added code that should be tested, supplement it with tests located in the `test/` directory.
+4. If you've added code that should be tested, supplement it with tests located in the `test/` directory (see [Adding a test](#adding-a-test)).
```

Comment:
Stronger wording should probably be used. 
Most code requires testing, cases were test don't have to be added is if the difference between the old and the new code is already visible in the test.